src/FSharp.Data.GraphQL/Execution.fs

```diff
@@ -470,4 +474,49 @@ let internal execute (schema: #ISchema) doc operationName variables root errors
 // we don't need to know possible types at this point
 SchemaMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> SchemaMetaFieldDef
 TypeMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> TypeMetaFieldDef
-TypeNameMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> TypeNameMetaFieldDef
\ No newline at end of file
+TypeNameMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> TypeNameMetaFieldDef
+
+
+type Define with
+    /// Single subscription defined inside either object types or interfaces 
+    static member Subscription(name : string, typedef : ObjectDef<'Res>,
+                               subscribe : ResolveFieldContext -> 'Val -> IObservable<NameValueLookup> -> unit,
+                               ?resolve: ResolveFieldContext -> 'Val -> IObservable<'Res>,
+                               ?description: string, ?args : InputFieldDef list, ?deprecationReason : string) : FieldDef<'Val> =
+        upcast { Name = name
+                 Description = description
+                 Type = typedef
+                 Resolve = fun ctx value -> job {
+                     let set =
```

Comment:
I think, that this kind of logic should not reside here. If necessary, it should be probably computed at the moment when Schema is created (just like `ExecuteInput` and `IntrospectionSchema`).